app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -231,6 +251,7 @@ class MessagingService : FirebaseMessagingService() {
                             NotificationActionReceiver.OPEN_URI
                         else
                             NotificationActionReceiver.FIRE_EVENT
+                    putExtra(NotificationActionReceiver.EXTRA_NOTIFICATION_TAG, tag)
```

Comment:
if this is null, will it override other null notification tags because we still marked it as the tag ?

Reply:
Nope, passing null as a tag is the same as if you call without a tag.